docs/advanced/errors.md

```diff
@@ -27,6 +27,86 @@ export class MyErrorHandler implements ErrorHandler {
 export class AppModule {}
 ```
 
+## Handling errors within an `@Select`
+
+```ts
+@Component({
+  /* .. */
+})
+class AppComponent {
+  @Select(state => state.count.number.value)
+  public count$: Observable<number>;
+}
+```
+
+One of the main problems is that if your application state is not correct later.
+
+```ts
+this.store.reset({}); // destroy all states
+```
+
+Then automatically all subscribers will be unsubscribed from the store, as an internal error will occur.
+To track such things, you must disable `suppressErrors` flag:
+
+```ts
+@NgModule({
+  imports: [
+    NgxsModule.forRoot([CountState], {
+      selectorOptions: {
+        suppressErrors: false // default by true
+      }
+    })
+  ]
+})
+export class AppModule {}
+```
+
+This way you can track application errors and add error handling.
+
+```ts
+@Component({
+  /* .. */
+})
+class AppComponent {
+  @Select(state => {
+    try {
+      return state.count.number.value;
+    } catch (err) {
+      console.log('error', err);
+      // throw err;
+      // If you forward an error further, then an automatic unsubscription from the stream will occur.
+      // Do not do this if you do not want it.
+    }
+  })
+  public count$: Observable<number>;
+}
+```
+
+#### Why does Rxjs unsubscribe on error?
```

Comment:
Seems like it's a little bit complicated information. Maybe we could just leave the link to this info? What do you think?

Reply:
done